### ISSUE TITLE
Dockercli.exe doesn't recognise -h or --help flags

### ISSUE THREAD

**Issue by pieterhouwen** [docker/for-win#11751]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID: N/A

### Actual behavior

```
PS C:\Program Files\Docker\Docker> .\DockerCli.exe -h
Usage: DockerCli.exe [Command]
  -h, --help, -Help: Show the help information for this command
PS C:\Program Files\Docker\Docker> .\DockerCli.exe --help
Usage: DockerCli.exe [Command]
  -h, --help, -Help: Show the help information for this command
PS C:\Program Files\Docker\Docker> .\DockerCli.exe --Help
PS C:\Program Files\Docker\Docker>
```

### Expected behavior

Some beautiful help screen or command line arguments should show.

### Information

  - Windows Version: 10 21H1 (build 19043.1110)
  - Docker Desktop Version: Checked for updates, nothing found, so I guess latest at time of writing
  - WSL2 or Hyper-V backend?  WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: Nope

### Steps to reproduce the behavior

1. Open CMD or Powershell.
2. Navigate to Docker Desktop folder
3. run Dockercli.exe -h/--help/--Help
4. Don't get a list of command-line args

**Comment by StefanScherer** [comment]
Hi @pieterhouwen
`DockerCli` is an internal tool that we never documented and will probably remove at some point in the future.

That being said if you explained a little what you’re looking to do, there might be other ways?

**Comment by pieterhouwen** [comment]
Hi Stefan,

After my graphics card failed I swapped it and reset my BIOS, thereby disabling CPU virtualisation which caused Docker to not start anymore even when I turned virtualization back on. When looking online for a solution to my error message (operation timed out) I came accross a post which recommended to switch daemons using the CLI tool.

After that didn't work I saw the next solution which seemded to work for people was to delete their docker folders in AppData which wrecked my Docker installation completely :) 

After a reinstall and reinstallation of my Docker images everything started working again so problem solved I guess?

But back to the original issue: If the cli tool won't get documentation and will be removed (probably) then I guess the issue can be closed.
